### ISSUE TITLE
Search form should have an aria-label

### ISSUE THREAD

**Issue by joshsmith01** [olefredrik/FoundationPress#1307]
**How can this bug be reproduced?**

1. Perform a Chrome Dev Tools Audit with at least Accessibility and SEO options selected
2. Accessibility audit result indicates that the Search form input could use a label

**What did you expect to happen?**
I expected the form input to have a label associated with it to pass Accessibility audits
**What happened instead?**
Accessibility audit of this particular item failed

**Please List the Following:**
* OS & version: Mac OS 10.13.6
* Node version (node -v) [Node v4+ is required] : v9.11.1
* FoundationPress version (see line 3 in package.json): 2.10.4
* Foundation version (found in `node_modules/foundation-sites/package.json`) : 6.4.3

**Include Test Case** (if applicable):  
Screenshot of failing item:
https://cl.ly/38ec95594f6d
